Makefile

```diff
@@ -105,4 +105,7 @@ format-in-docker:
 
 .PHONY: all-in-docker
 all-in-docker:
-	make in-docker ARGS="make all"
\ No newline at end of file
+	make in-docker ARGS="make all"
+#########################################Build onomy load test binary#########################
+build-onomy-load-test:
```

Comment:
add please the build tag "tmload" to all new go files, to the make build command, and linters. Similar like id did in wnom test with "integration" tag. That will help not to include those binaries if we don't need them.